# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/systemservices/objc_video_capture/video_capture_avfoundation.mm
# Commit: 0fa64a630cc7
# Full Hash: 0fa64a630cc72eec221fbdc122250643fd9dd97f
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2023-01-11 21:56:57
# Regressor Bug: 1806605
# File Overlap Count: 1
# Description:
#   Bug 1806605 - Fix PerformanceRecorder not adding markers in VideoCaptureAvFoundation. r=padenot
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D165823
# ==============================================================================

diff -r 952d58ad6573 -r 0fa64a630cc7 dom/media/systemservices/objc_video_capture/video_capture_avfoundation.mm
--- a/dom/media/systemservices/objc_video_capture/video_capture_avfoundation.mm	Wed Jan 11 11:47:44 2023 +0000
+++ b/dom/media/systemservices/objc_video_capture/video_capture_avfoundation.mm	Wed Jan 11 11:47:45 2023 +0000
@@ -237,7 +237,7 @@
 void VideoCaptureAvFoundation::StartFrameRecording(int32_t aWidth, int32_t aHeight) {
   MaybeRegisterCallbackThread();
   MutexLock lock(&api_lock_);
-  if (MOZ_UNLIKELY(mTrackingId)) {
+  if (MOZ_UNLIKELY(!mTrackingId)) {
     return;
   }
   auto fromWebrtcVideoType = [](webrtc::VideoType aType) -> CaptureStage::ImageType {
